internal/namespaces/registry/v1/custom_docker_helper.go

```diff
@@ -0,0 +1,255 @@
+	helperScript := fmt.Sprintf("#!/bin/sh\n\n%s%s registry docker-helper \"$@\"", binaryName, profileFlag)
+
+	// Warning
+	_, _ = interactive.Println()
+	_, _ = interactive.PrintlnWithoutIndent("To enable the Docker credential helper we need to create the file " + helperScriptPath + " with the following lines:")
+	_, _ = interactive.Println(helperScript)
+
+	// Early exit if user disagrees
+	_, _ = interactive.Println()
+	continueInstallation, err := interactive.PromptBoolWithConfig(&interactive.PromptBoolConfig{
+		Prompt:       fmt.Sprintf("Do you want to proceed with these changes?"),
+		DefaultValue: true,
+	})
+	if err != nil {
+		return nil, err
+	}
+	if !continueInstallation {
+		return nil, fmt.Errorf("installation cancelled")
+	}
+
+	err = writeHelperScript(helperScriptPath, helperScript)
+	if err != nil {
+		return nil, fmt.Errorf("failed to write helper script: %s", err)
+	}
+
+	var registries []string
+	for _, region := range scw.AllRegions {
+		registries = append(registries, endpointPrefix+region.String()+endpointSuffix)
+	}
+
+	err = setupDockerConfigFile(registries, binaryName)
+	if err != nil {
+		return nil, fmt.Errorf("failed to write docker config file: %s", err)
+	}
+
+	_, _ = interactive.Println()
+	_, err = exec.LookPath(fmt.Sprintf("docker-credential-%s", binaryName))
+	if err != nil {
+		_, _ = interactive.Println(fmt.Sprintf("docker-credential-%s is not present in your $PATH, you should add %s to your $PATH to make it work.", binaryName, path.Dir(helperScriptPath)))
+		_, _ = interactive.Println(fmt.Sprintf("You can add it by adding `export PATH=$PATH:%s` to your `.bashrc` or `.zshrc`", path.Dir(helperScriptPath)))
+	} else {
+		_, _ = interactive.PrintlnWithoutIndent("Docker credential helper successfully installed.")
+		_, _ = interactive.PrintlnWithoutIndent("You can now pull/push without logging in to your registries.")
+	}
+
+	return nil, nil
+}
+
+type registryDockerHelperGetResponse struct {
+	Secret   string `json:"Secret"`
+	Username string `json:"Username"`
+}
+
+func registryDockerHelperGetCommand() *core.Command {
+	return &core.Command{
+		Hidden:    true,
+		Namespace: "registry",
+		Resource:  "docker-helper",
+		Verb:      "get",
+		ArgsType:  reflect.TypeOf(emptyRequest{}),
+		ArgSpecs: []*core.ArgSpec{
+			{},
+		},
+		Run:         registryDockerHelperGetRun,
+		PrinterType: &printer.JSON,
+	}
+}
+
+func registryDockerHelperGetRun(ctx context.Context, argsI interface{}) (i interface{}, e error) {
+	in := bufio.NewReader(os.Stdin)
+	var serverURL string
+	for {
+		s, err := in.ReadString('\n')
+		if err != nil {
+			if err != io.EOF {
+				return nil, err
+			}
+			serverURL = serverURL + s
+			break
+		}
+		serverURL = serverURL + s
+	}
+
+	serverURL = strings.TrimSpace(serverURL)
+
+	serverFound := false
+
+	for _, region := range scw.AllRegions {
+		if serverURL == endpointPrefix+region.String()+endpointSuffix {
```

Comment:
I think there is a methiod for that on master

Reply:
on #911 ?